src/main/java/stirling/software/SPDF/config/security/AppUpdateShowService.java

```diff
@@ -0,0 +1,46 @@
+package stirling.software.SPDF.config.security;
+
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+
+import stirling.software.SPDF.model.ApplicationProperties;
+import stirling.software.SPDF.model.User;
+import stirling.software.SPDF.repository.UserRepository;
+
+@Service
+public class AppUpdateShowService {
```

Comment:
Having this in the security folder means non security enabled jars will have build issues

Reply:
Also adding it as parameter to all view files isnt a good solution personally i prefer doing it via bean such as the global properties in
https://github.com/Stirling-Tools/Stirling-PDF/blob/main/src/main/java/stirling/software/SPDF/config/AppConfig.java

Reply:
in `AppConfig.java` I cannot query the user's status, as in the case of whether he is an admin or not

Reply:
I dont mean placing in AppConfig but that @Bean(name = "appName") is a nicer way

Reply:
Ok I will try to use it 😅

Reply:
```bash

 ./gradlew  compileJava

> Configure project :
0.22.1

BUILD SUCCESSFUL in 6s
4 actionable tasks: 2 executed, 2 up-to-date
```